MODULE.bazel

```diff
@@ -0,0 +1,12 @@
+"bazel-contrib/rules_oci"
+
+module(
+    name = "contrib_rules_oci",
+    compatibility_level = 1,
+    version = "0.0.0",
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.3.0")
```

Comment:
Version 1.4.1 is available as of 9 February 2023, per your merge of bazelbuild/bazel-central-registry#435.